packages.yml

```
@@ -1,3 +1,3 @@
 packages:
 - package: fivetran/fivetran_utils
-  version: [">=0.1.0", "<0.2.0"]
+  version: [">=0.1.0", "<0.3.0"]
```

```suggestion
  version: [">=0.2.0", "<0.3.0"]
```
With this most recent update we are going to need to restrict the range of fivetran_utils to the v0.2.x release(s). This is due to the recent fivetran_utils release points to dbt-labs and the older versions pointing to fishtown-analytics.